src/supertux/menu/addon_menu.cpp

```diff
@@ -121,13 +121,13 @@ AddonMenu::rebuild_menu()
   {
     const Addon& addon = m_addon_manager.get_installed_addon(m_installed_addons[index]);
     const std::string text = addon_string_util::generate_menu_item_text(addon);
-    add_entry(MAKE_UPDATE_MENU_ID(index), fmt::format(fmt::runtime(_("{} {}*UPDATE*")), text, !addon.is_enabled() ? "[DISABLED] " : ""));
+    add_entry(MAKE_UPDATE_MENU_ID(index), fmt::format(fmt::runtime(_("{} {}*UPDATE*")), text, !addon.is_enabled() ? (_("[DISABLED]") + " ") : ""));
   }
   for (const auto& index : addons_to_list)
   {
     const Addon& addon = m_addon_manager.get_installed_addon(m_installed_addons[index]);
     const std::string text = addon_string_util::generate_menu_item_text(addon);
-    add_entry(MAKE_INSTALLED_MENU_ID(index), fmt::format(fmt::runtime(_("{}{}")), text, !addon.is_enabled() ? " [DISABLED]" : ""));
+    add_entry(MAKE_INSTALLED_MENU_ID(index), fmt::format(fmt::runtime(_("{}{}")), text, !addon.is_enabled() ? (" " + _("[DISABLED]")) : ""));
```

Comment:
The `_("{}{}")`, which was already here before the changes, looks wrong to me. Does it translate the string `{}{}` before substitutions are applied?

Reply:
It doesn't make sense for that to have translation markers. It doesn't have any translatable content.

Reply:
I'll fix it tomorrow.

Reply:
Nitpick question: Is the `fmt::runtime` still needed if the string is known at compile time?

Reply:
Honestly, it should be "{} [disabled]" as translated string, because some rtl locales need the add-on name on the right.

Reply:
Honestly, it should be "{} [disabled]" as translated string, because some rtl locales need the add-on name on the right.